cli/command/utils.go

```diff
@@ -16,3 +18,21 @@ func extractArgs(received []string, names ...string) (map[string]string, error)
 
 	return args, nil
 }
+
+func buildQueryHelper(id, start, end string, tail int) url.Values {
+	query := url.Values{}
+
+	if tail > 0 {
+		query.Set("tail", strconv.Itoa(tail))
```

Comment:
use the `const`s in the `client` package for these keys:
```
query.Set(client.LogQueryParamTail, strconv.Itoa(tail))
```
see: https://github.com/quintilesims/layer0/blob/cli-refactor/client/helpers.go#L19